airflow/providers/greatexpectations/operators/greatexpectations_base.py

```diff
@@ -0,0 +1,30 @@
+from airflow.models import BaseOperator
+
+
+class GreatExpectationsBaseOperator(BaseOperator):
```

Comment:
@turbaszek , this is the new GreatExpectationsBaseOperator.  It's really just an interface right now.  That was the approach recommended by the GE team because they are planning to do a big refactor and it's not worth trying to move any generic functionality into the base operator until the refactor is done.

Reply:
Then I would say there's no need for such an operator - it gives us nothing currently. We should add the operator / hook only if it brings some value, for example creates repository of reusable methods. In this case, this may only make future refactoring harder.

Reply:
> the base operator could implement a generic load data context + load batch + validation

I agree with that, the base operator should implement methods that can be reused in other more specific operators. It should also implement some error handling as well as auth part if necessary